modules/lakeformation/main.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_lakeformation_permissions" "segment_data_lake_formation_table_emr" {
+  principal = var.emr_role
+  permissions = ["ALL"]
+
+  table {
+    database_name = var.name
+    wildcard = true
+  }
+}
+
+resource "aws_lakeformation_permissions" "segment_data_lake_formation_database_emr" {
```

Comment:
Is there a reason to name these resources `segment_data_lake_formation_...`? Or did you meant to have something like `segment_data_lake_lake_formation_...` (i.e. Segment Datalake Lake Formation)?